### PR TITLE
[#136061709] Improve export options

### PR DESCRIPTION
**Pull request by yl5682**
As a scholarly edition author I can use a dynamic form that breaks the export process up into logical steps so that I understand the choices I am making when exporting a volume and avoid making incompatible choices.

https://www.pivotaltracker.com/story/show/136061709

Please help me review if there is any space for improvement. Thanks!

### CODE REVIEW

File: readux/books/templates/books/volume_export.html [hunk 1]
```diff
@@ -124,46 +127,77 @@
...
+          <button disabled type="submit" class="btn btn-primary center-block export-btn" id="export-button"><i class="fa fa-download" aria-hidden="true"></i> Export</button>
         </div>
     </form>
 
-
     <script>
     {# Show/hide subportions of the form based on export mode #}
       $(document).ready(function(){
-        var transition = 500;
+        var transition = 300;
```
Comment (jayvarner) [inline readux/books/templates/books/volume_export.html]: Are there tests for this? If not, we should add a chore to write them.

File: readux/books/templates/books/volume_export.html [hunk 2]
```diff
@@ -287,6 +321,27 @@
       });
     </script>
 
+
+    <style>
```
Comment (jayvarner) [inline readux/books/templates/books/volume_export.html]: Is there a reason you included the CSS in the html and not in the external file.

File: readux/books/templates/books/volume_export.html [hunk 3]
```diff
@@ -124,46 +127,77 @@
...
+            <label>
+              {{ field }} {{ field.label }}
+            </label>
             <div class="help-block">{{ field.help_text}}</div>
-            <div class="text-warning {{ field.name }}-errors">{{ field.errors }}</div>
-            {# TODO: display validation errors, required/optional ? #}
-        </div>
+          </div>
+        {% else %}
+          <div style="display:none" class="form-group export-form-group{% if field.errors %} has-error{% endif %} {% if field.name in export_form.jekyll_options %}jekyll-only{% endif %}"{% if field.name = 'github_repo' or field.name = 'update_repo' %}style="display:none"{% endif %}>
```
Comment (jayvarner) [inline readux/books/templates/books/volume_export.html]: I like to avoid inline style.

Comment (yl5682) [inline readux/books/templates/books/volume_export.html]: Moved to its own CSS and created appropriate classes.

File: readux/books/templates/books/volume_export.html (hunk 2 above)
Comment (yl5682) [inline readux/books/templates/books/volume_export.html]: Moved to its own CSS.

**Pull request merged**